data/data/aws/vpc/vpc.tf

```diff
@@ -1,6 +1,6 @@
 locals {
-  new_worker_cidr_range = "${cidrsubnet(data.aws_vpc.cluster_vpc.cidr_block,1,1)}"
-  new_master_cidr_range = "${cidrsubnet(data.aws_vpc.cluster_vpc.cidr_block,1,0)}"
+  new_compute_cidr_range = "${cidrsubnet(data.aws_vpc.cluster_vpc.cidr_block,1,1)}"
```

Comment:
You need to [format your Terraform][1].

[1]: https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_installer/1033/pull-ci-openshift-installer-tf-fmt/2475/build-log.txt